### PR TITLE
Change name to use react-package convention

### PR DESCRIPTION
**Pull request by bassjacob**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,12 +1,11 @@
 {
   "name": "react-filterbar",
-  "version": "0.0.0",
```
Comment (jeremygrant) [inline bower.json]: should we maybe add a version instead of completely removing it?

Comment (subshad) [inline bower.json]: bower doesn't use this version key for anything. It looks at git tags.

**Comment by jeremygrant** [comment]
:+1:

File: bower.json (hunk 1 above)
Comment (bradb) [inline bower.json:3]: This is slightly outside the boundaries of my knowledge, but I agree with @jeremygrant - a version string seems necessary here. I know nothing about how bower uses version strings, but they are documented here http://bower.io/docs/creating-packages/, and it seems like, e.g. they can be used to specify dependencies. (And in general, giving a package a version number seems better than giving it no version number.)

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "react-filterbar",
   "version": "0.0.0",
   "description": "",
-  "main": "dist/reactFilterBar.js",
+  "main": "dist/react-filterbar.js",
   "dependencies": {
     "bootstrap": "^3.3.4",
     "jquery": "^2.1.3",
```
Comment (bradb) [inline package.json:8]: That's all I got. :octocat:

File: bower.json (hunk 1 above)
Comment (bassjacob) [inline bower.json:3]: I'll throw up a PR adding it in, and suggest keeping it in line with the git tag. Bower seems to ignore the bower.json version key for the near future, at least until they get a proper package server.

**Pull request merged**